modules/www/main.tf

```
@@ -407,28 +422,29 @@ resource "aws_ecs_task_definition" "www_task_definition" {
        "privileged": false,
        "name": "nerves_hub_www",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         },
-         {
-           "name": "APP_NAME",
-           "value": "nerves_hub_www"
-         }
-       ],
-       "logConfiguration": {
-         "logDriver": "awslogs",
-         "options": {
-           "awslogs-region": "${var.region}",
-           "awslogs-group": "${var.cluster.log_group_name}",
-           "awslogs-stream-prefix": "nerves_hub_www"
-         }
-       }
-     }
+         ${local.ecs_shared_env_vars}
+       ]
```

If we remove the "mountPoint" and "volumesFrom" in the other PR, we can add this and it should work. 
```suggestion
       ],
       ${module.firelens_log_config.log_configuration}
```

---

removed